docs/DEVELOPER-ADVANCED.md

```diff
@@ -24,7 +24,10 @@ To get your Ubuntu 16.04 or 18.04 LTS install up and running to develop Discours
 
     rvm install 2.6.2
     rvm --default use 2.6.2 # If this error out check https://rvm.io/integration/gnome-terminal
-    gem install bundler mailcatcher rake
+    gem install bundler rake
+
+    #MailHog (https://github.com/mailhog/MailHog)
+    go get github.com/mailhog/MailHog
```

Comment:
Similar to my [comment in discourse_docker](https://github.com/discourse/discourse_docker/pull/572#discussion_r721611265), I don't think this makes `mailhog` available on the PATH. We might want to move the binary to somewhere on the PATH (and maybe rename it to `mailhog`? Uppercase characters in the binary name seems a little weird)